packages/design-system/src/styles/components/_VerticalNav.scss

```diff
@@ -98,9 +98,9 @@
   }
 
   .ds-c-icon {
+    aspect-ratio: 1;
     color: var(--vertical-nav-label-icon__color);
-    height: var(--font-size-md);
-    width: var(--font-size-md);
+    width: var(--font-size-md, 1em);
```

Comment:
Good working getting to the bottom of what causes the flash. What I don't get is why it wouldn't have this value. The theme file with this variable is in the `<head>` of the document, and all stylesheets loaded in the head are supposed to block rendering.

Reply:
good point - could this be how gatsby's wired up?

when viewing the sources tab in chrome dev tools, i see 2 potentials sources for css: one inlined and tied to the url `?theme=X`, which gets loaded as the html is loaded, and another pointing to a linked stylesheet listing all css custom prop values for the same theme, which loads after the html/inline styles (attaching images below). 

i'm not sure where this logic comes into play, but our docs site seems to split our token files apart from our styles.

<img width="793" alt="css custom property values for cmsgov" src="https://github.com/CMSgov/design-system/assets/5159392/1fae163b-638e-42c2-b4f1-e421b3c2ea76">

<img width="883" alt="inlined css for cmsgov theme" src="https://github.com/CMSgov/design-system/assets/5159392/5f011406-a8d6-40c3-a652-e315c9034e3e">

Reply:
The styles that are inlined for `?theme=cmsgov` are the universal styles imported by `docs/src/styles/index.scss`, and they don't contain any theme data. It's the equivalent of loading our `index.css`. I took a look at one of the statically built html files in our docs `public` folder after running `yarn build:docs`. Almost right after the inline styles in the `<head>` comes a `<link>` to the core theme stylesheet (which gets replaced when JavaScript loads). Since it has the core stylesheet loaded, it should have values for those variables. Could there possibly be a moment when `Helmet` _removes_ the core stylesheet and _adds_ the selected theme's stylesheet and in that moment there are no theme variables?